tests/installation/installation_overview.pm

```diff
@@ -62,6 +62,30 @@ sub ensure_ssh_unblocked {
     }
 }
 
+sub check_default_target {
+    # Check the systemd target where scenario make it possible
+    return if (is_caasp || is_upgrade || is_hyperv ||
+        get_var('REMOTE_CONTROLLER') || (get_var('BACKEND', '') =~ /spvm|pvm_hmc/));
+    # exclude non-desktop environment as well
+    return unless get_var('DESKTOP');
+
+    # Set expectations
+    my $expected_target = check_var('DESKTOP', 'textmode') ? "multi-user" : "graphical";
+
+    select_console 'install-shell';
+
+    # default.target is not yet linked, so we have to parse the logs
+    my $target_search = 'default target has been set';
+    my $log_line      = script_output("grep '$target_search' /var/log/YaST2/y2log | tail -1");
+
+    # Assert expectations
+    die "Expected default.target '$expected_target' not been detected in YaST logs" unless $log_line;
```

Comment:
Tbh, I would skip the check in case we were not able to detect expected target, not to break tests in case there is something wrong with code. WDYT?

Reply:
agreed, make sense to relax expectations.